### DISCUSSION TITLE
"Error: No available formula with the name "sdkman-cli". Did you mean sigma-cli?" Keeps coming up on terminal startup.

### DISCUSSION THREAD

**Discussion by suniljose25** [Homebrew/discussions#5162] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: c6d959218f143cd17b1fc3e0f10f143cbd273528
Last commit: 2 days ago
Core tap JSON: 28 Feb 02:31 UTC
Core cask tap JSON: 28 Feb 02:31 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
HOMEBREW_NO_ANALYTICS: set
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.3.1-x86_64
CLT: 15.1.0.0.1.1700200546
Xcode: 15.2
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1
```

### Description of issue

I had previously added a tap "sdkman/tap" to try to install "sdkman-cli" formula. However after "sdkman" did not work as intended, I removed both the "sdkman-cli" formula and the "sdkman/tap" tap.

Since untapping "sdkman/tap", this error has been coming up every time I open a new terminal
`Error: No available formula with the name "sdkman-cli". Did you mean sigma-cli?`

I have tried `brew update`, `brew upgrade`, and `brew cleanup`.

Do you know what went wrong here? Do you know how I can fix this and stop this error from showing up?

**Comment by SMillerDev** [comment]
Did you put something in your `~/.bashrc` or `~/.zprofile` file about sdkman?

**Reply by suniljose25** [reply]
Yup, it seems that there was a script in `~/.zprofile` which was doing `brew --prefix sdkman-cli`. Removing that fixed the problem.

**Comment by gromgit** [comment, marked as answer]
> Since untapping "sdkman/tap", this error has been coming up every time I open a new terminal `Error: No available formula with the name "sdkman-cli". Did you mean sigma-cli?`

That seems to point to a `brew` command in your shell startup that's trying to find or install `sdkman-cli`. To find it, do one of the following to log your shell's startup sequence, depending on which one you're using:
* **Zsh**: `zsh -xli </dev/null 2>&1 | tee /tmp/shellstart.log`
* **Bash**: `exec 7>/tmp/shellstart.log ; PS4='+${BASH_SOURCE}:${LINENO}> ' BASH_XTRACEFD=7 bash -xli </dev/null ; exec 7>&-`

In either case, search `/tmp/shellstart.log` for the error message, then go up one or more lines to see what command triggered it, as well as which file/line it can be found in.
